snow/consensus/snowman/consensus_test.go

```diff
@@ -107,7 +107,7 @@ func InitializeTest(t *testing.T, factory Factory) {
 
 	if pref := sm.Preference(); pref != GenesisID {
 		t.Fatalf("Wrong preference returned")
-	} else if !sm.Finalized() {
+	} else if !(sm.NumProcessing() == 1) {
```

Comment:
(this comment applies to other places)
```suggestion
	} else if sm.NumProcessing() > 0 {
```

An instance is finalized if there are no processing blocks.